source/vir/src/ast_to_sst.rs

```diff
@@ -21,6 +21,9 @@ type Arg = (Exp, Typ);
 type Args = Arc<Vec<Arg>>;
 
 pub(crate) struct State {
+    // View exec/proof code as spec
+    // (used for is_const functions, where we let Rust --compile check for overflow)
```

Comment:
Not a huge fan of the idea of relying on `--compile` for stuff like this. (If I'm a developer, I'd want to get an error for this early, before trying to verify everything else.) 

Are there other things we rely on `--compile` for? Could we make at least make a TODO item to handle this in VIR?

Reply:
Ok, this is now fixed.